src/operators/skipUntil.ts

```diff
@@ -19,8 +19,9 @@ class SkipUntilOperator<T, R> implements Operator<T, R> {
 class SkipUntilSubscriber<T> extends Subscriber<T> {
   private notificationSubscriber: NotificationSubscriber<any> = null;
 
-  constructor(destination: Subscriber<T>, private notifier: Observable<any>) {
-    super(destination);
+  constructor(public destination: Subscriber<T>,
+              private notifier: Observable<any>) {
+    super(null);
```

Comment:
I see what you're saying. It makes me wonder what other operators may have this side-effect from the single composite subscription architecture. @trxcllnt, what do you think?

It seems to me like, at least for `skipUntil` we can assign the `destination` in the subscriber directly rather than going through the `super` call. (As is in this PR). It does concern me that this will somehow break the unsubscription chain in cases where there is more than one operator before  `skipUntil`, though. I haven't put much thought into that.